### ISSUE TITLE
Compression error causes performance drop

### ISSUE THREAD

**Issue by erow** [libffcv/ffcv#364]
Hi, I found a problem of FFCV, which may not the fault of the library, but this library suffers from it.

I use the same model to evaluate the accuracy on different validation sets. The results show compression reduces the performance significantly, but it's too much for FFCV. 
![image](https://github.com/libffcv/ffcv/assets/7524356/cf954f4e-813f-4dc4-94d7-168e3236d9ba)

I applied FFCV to accelerate SSL. The performance dropped for both pretraining and finetuning stages.

We set the JPEG compression quality at 90.0% to maintain high visual fidelity, write mode to jpg, the maximal resolution to 1000.

First, I hope that a lossless compression is supported.
Second, I wonder any solution to reduce the compression shidt.
